timesketch/ui/graphs/cytoscape.component.spec.ts

```diff
@@ -0,0 +1,264 @@
+import {Component, ViewChild} from '@angular/core'
+import {TestBed, ComponentFixture} from '@angular/core/testing'
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting} from '@angular/platform-browser-dynamic/testing'
+import {BrowserModule} from '@angular/platform-browser'
+
+import {CytoscapeComponent} from './cytoscape.component'
+
+export const testCytoscapeOptions = {
+  zoomingEnabled: false,
+  userZoomingEnabled: false,
+  panningEnabled: false,
+  userPanningEnabled: false,
+  boxSelectionEnabled: true,
+  autolock: true,
+  autoungrabify: true,
+  autounselectify: true,
+  headless: true,
+  hideEdgesOnViewport: true,
+  hideLabelsOnViewport: true,
+  textureOnViewport: true,
+  motionBlur: true,
+  styleEnabled: false,
+  minZoom: 1e-10,
+  maxZoom: 1e10,
+  touchTapThreshold: 10,
+  desktopTapThreshold: 6,
+  motionBlurOpacity: 0.4,
+  wheelSensitivity: 0.8,
+  pixelRatio: 1.0,
+  selectionType: 'additive',
+}
+
+export const getOptionFromCytoscape = (_cy: Cy.Core, option: string) => {
+  const cy = _cy as any
+  if (option in cy)
```

Comment:
I would either add braces here or write the return on the same line as the if.

Reply:
Ok.

Reply:
Done. TSLint rule added.